QUDAConfig.cmake.in

```diff
@@ -7,6 +7,9 @@ set(QUDA_MPI @QUDA_MPI@)
 set(QUDA_QIO @QUDA_QIO@)
 set(QUDA_OPENMP @QUDA_OPENMP@)
 set(QUDA_QDPJIT @QUDA_QDPJIT@)
+set(QUDA_GITVERSION @GITVERSION@)
```

Comment:
Where is this being used then? Mostly curious.

Reply:
The tuning uses the GITVERSION as a flag to declare when tunecache is obsolete. I then use it as well to create tuning directories that are dependent on it (unless the user specify its own `QUDA_RESOURCE_PATH`).